### DISCUSSION TITLE
Hardware Compatability

### DISCUSSION THREAD

**Discussion by jramborger78** [talmolab/sleap#1545] [Help!]
Hi all!

Thank you for always being so helpful, I had a question about device choses.

I could not find any direct mentions of suggestions other than a CUDA core compatible 8GB GPU. I reviewed the Hardware Requirements online at the DeepLabCut website given its similarities to SLEAP and just wanted to pop in and see if my device is actually suitable. According to benchmarks this GPU falls in between the DeepLabCut suggested 1080 Ti and 2080 Ti so I am leaning towards yes but wanted to maybe check:

Dell XPS 15 (laptop)
- 13th Gen Intel® Core™ i9-13900H (24 MB cache, 14 cores, up to 5.40 GHz Turbo)
- NVIDIA® GeForce RTX™ 4070, 8 GB GDDR6
- 64 GB: 2 x 32 GB, DDR5, 4800 MT/s, dual-channel RAM

Thanks!

**Comment by roomrys** [comment]
Hi @jramborger78,

Sorry for the late reply. My initial reaction was total envy - the XPS is top notch. But, I wanted to double-check/investigate all the components to make sure I didn't give you bad advice. The results: yes, the XPS is top notch. The only thing to maybe be aware of is that 8 GB of GDDR6 VRAM would be the weakest link here (not that it's weak). Very larger datasets with lots of labels and very high resolution video (e.g. 8K with 7680 x 4320 pixels per frame) may cause Out of Memory issues, but in general it should be fine.

Thanks,
Liezl

**Comment by jramborger78** [comment]
Not a problem! Love that though, thank you so much! Ours are shot in 8MP on the Raspberry Pi v2 at 1280x720. Sounds like it should be good to go!

Sincerely,
Jarryd Ramborger
